Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -0,0 +1,2006 @@
+/*
+ * Copyright (c) 2023, Lucas Chollet <lucas.chollet@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/BitStream.h>
+#include <AK/Endian.h>
+#include <AK/FixedArray.h>
+#include <AK/String.h>
+#include <LibGfx/ImageFormats/JPEGXLLoader.h>
+
+namespace Gfx {
+
```

Comment:
having a link to a spec doc up here would be nice

Reply:
As the spec is closed, no public link is accessible. It's not solved, but there is nothing we can't do about it.